cli/CHANGELOG.md

```diff
@@ -1,16 +1,18 @@
 <!-- See the ../guides/writing-the-cypress-changelog.md for details on writing the changelog. -->
 ## 13.7.0
 
-_Released 3/12/2024 (PENDING)_
+_Released 3/13/2024_
 
 **Features:**
 
 - Added shadow DOM snapshot support within Test Replay in order to highlight elements correctly within the Cypress reporter. Addressed in [#28823](https://github.com/cypress-io/cypress/pull/28823).
 - Added TypeScript support for [Vue 2.7+](https://github.com/vuejs/vue/blob/main/CHANGELOG.md#270-2022-07-01). Addresses [#28591](https://github.com/cypress-io/cypress/issues/28591).
+- Adds additional context to error messages displayed when Test Replay artifacts fail to upload. Addressed in [#28986](https://github.com/cypress-io/cypress/pull/28986)
 
 **Performance:**
 
-- Fixed a performance regression from [`13.6.4`](https://docs.cypress.io/guides/references/changelog#13-6-4) where unhandled service worker requests may not correlate correctly. Fixes [#28868](https://github.com/cypress-io/cypress/issues/28868).
+- Fixed a performance regression from [`13.6.3`](https://docs.cypress.io/guides/references/changelog#13-6-3) where unhandled service worker requests may not correlate correctly. Fixes [#28868](https://github.com/cypress-io/cypress/issues/28868).
```

Comment:
wasn't the performance regression introduced in 13.6.4? From what I've tested, 13.6.3 works fine and from 13.6.4 on it doubles the time for network responses in my tests.

Reply:
@gabrielbahniuk, not that we are aware of. Can you try the pre-release binary (or wait until 13.7.0 is released) to see if it resolves your performance issue:

```
npm install https://cdn.cypress.io/beta/npm/13.7.0/linux-x64/develop-f8f7e71907870a50784a1a8547940902362b58e2/cypress.tgz
```

If your performance issue is not resolved, could you please open a new issue with a reproducible example so we can investigate.